docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -2,9 +2,12 @@
 Submitting contributions
 ========================
 
-We're always grateful for contributions to Django's code. Indeed, bug reports
-with associated contributions will get fixed *far* more quickly than those
-without a solution.
+Thank you for your interest in contributing to Django's code.
+Django is run on the effort of volunteers and these documentations
+are to help you understand Django's processes and land your contribution
+successfully.
```

Comment:
I think we haven't got the wording of this quite right (I appreciate I suggested this wording). 

I am now thinking, this is still true. I don't think this contradicts the advice of needing a ticket. This is a bug report with a solution (ie. someone raised a ticket in trac **and** raised a pull request). Although this is not yet accepted, when someone is trying to fix a bug, it is great to see the suggested fix (especially for regressions). This happens already quite well. 

So perhaps let's leave this paragraph as it was initially. @nessita what do you think?

Reply:
@sarahboyce should be this? 

> We're always grateful for contributions to Django's code. Indeed, bug reports
> with associated contributions will get fixed *far* more quickly than those
> without a solution.

Reply:
Yes please